charts/lagoon-core/templates/api.deployment.yaml

```diff
@@ -68,6 +68,14 @@ spec:
           {{- end }}
         - name: REDIS_HOST
           value: {{ include "lagoon-core.apiRedis.fullname" . }}
+        - name: SSH_TOKEN_ENDPOINT
+          value: {{ .Values.sshTokenEndpoint | quote }}
```

Comment:
i think this will fail unless it is defined. but the default value is commented out?

Reply:
yuup - wrapped in an "if" now